qemu/tests/virtio_serial_throttling.py

```diff
@@ -0,0 +1,54 @@
+from virttest import error_context, env_process
+from virttest.utils_virtio_port import VirtioPortTest
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Induce throttling by opening chardev but not reading data from it:
+    1) Start guest with virtio-serial with unix option
+    2) On the host, open the chardev but don't read from it
+    3) In the guest, write to the virtio-serial port.
+    4) transfer data from host to guest but do not read in guest
+    5) repeat step 1 to 4 with tcp option.
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def send_data_from_guest_to_host():
+        session = vm.wait_for_login()
+        port.open()
+        session.cmd_output('dd if=/dev/zero of=/dev/virtio-ports/%s bs=1024 count=1' % port.name)
```

Comment:
we do not need to check the out put. Please use `session.cmd` ,  it will raise exception when the exit status is non zero.